src/FEValues/cell_values.jl

```diff
@@ -1,17 +1,20 @@
 """
-    CellValues([::Type{T}], quad_rule::QuadratureRule, func_interpol::Interpolation, [geom_interpol::Interpolation])
+    CellValues([sdim::Int,] [::Type{T},] quad_rule::QuadratureRule, func_interpol::Interpolation, [geom_interpol::Interpolation])
```

Comment:
This is not a very satisfying way of specifying the space dimension, but I am not sure what would be better. Perhaps requiring passing the vectorized geometric interpolation would be an idea?

```
qr = QuadratureRule{RefTriangle}(1)
ip = Lagrange{RefTriange, 1}()
ipg = Lagrange{RefTriangle, 1}() ^ sdim

CellValues(qr, ip, ipg)
```
and extracting `sdim` from `ipg` while unwrapping it to use the scalar versions, still, internally. Thoughts?

Reply:
That is what I wanted to do in the first place, but I also wanted to avoid having these weird inconsistencies that we pass the scalar geometric interpolation at some points and the vectorized at other points.